ReadMe.md

```diff
@@ -13,7 +13,7 @@ You can see all the packages and differences in the table below:
 | Package name                                | Latest Version | Content                                                                                                                | Usage Instructions                                                          |
 |---------------------------------------------|----------------|------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------|
 | LibMtpSharp                                 | 0.1.3          | Contains the wrapper managed code. API coverage is limited and will improve in future versions.                        | Main package with logic. Required to be able to use libmpt                  |
-| LibMtpSharpStandardMacOS                    | 0.1.6          | Contains the wrapper managed code with MacOS dependencies for .Net standard                                            | Main package with logic for .Net standard when you target MacOS             |
+| LibMtpSharpStandardMacOS                    | 0.1.8          | Contains the wrapper managed code with MacOS dependencies for .Net standard                                            | Main package with logic for .Net standard when you target MacOS             |
```

Comment:
Why 0.1.8 instead of 0.1.7?

Reply:
I added 0.1.8 version because 0.1.7 has bug in code (I detected this bug after publishing my 0.1.7 version)